### ISSUE TITLE
Inventory is always empty in getSpaceElevator

### ISSUE THREAD

**Issue by derpierre65** [porisius/FicsitRemoteMonitoring#91]
Inventory of the space elevator is always empty, but my inventory is not empty
![grafik](https://github.com/user-attachments/assets/02898940-25bc-4c34-b8fc-c736f89c98df)

```json
[
  {
    "Name": "Space Elevator",
    "ClassName": "Build_SpaceElevator_C",
    "location": {
      "x": -57599.1953125,
      "y": 266728.65625,
      "z": -3798.14697265625,
      "rotation": 89.57738174282781
    },
    "Inventory": [],
    "PhaseCost": [
      {
        "Name": "Assembly Director System",
        "ClassName": "Desc_SpaceElevatorPart_7_C",
        "Amount": 500
      },
      {
        "Name": "Magnetic Field Generator",
        "ClassName": "Desc_SpaceElevatorPart_6_C",
        "Amount": 500
      },
      {
        "Name": "Thermal Propulsion Rocket",
        "ClassName": "Desc_SpaceElevatorPart_8_C",
        "Amount": 250
      },
      {
        "Name": "Nuclear Pasta",
        "ClassName": "Desc_SpaceElevatorPart_9_C",
        "Amount": 100
      }
    ],
    "FullyUpgraded": false,
    "UpgradeReady": false,
    "features": {
      "properties": {
        "name": "Space Elevator",
        "type": "Space Elevator"
      },
      "geometry": {
        "coordinates": {
          "x": -57599.1953125,
          "y": 266728.65625,
          "z": -3798.14697265625
        },
        "type": "Point"
      }
    }
  }
]
```

**Comment by porisius** [comment]
Just so you know you are not forgotten, still looking for the inventory for AFGBuildableSpaceElevator as apparently mInputInventory is not used, or not used as we would expect.

I might have to do an asset dump of the game to know more.

**Comment by porisius** [comment]
Inventory is going to be dropped, there is literally no use for it that I can find.

May I have your feedback on this?

    "CurrentPhase": [
      {
        "Name": "Assembly Director System",
        "ClassName": "Desc_SpaceElevatorPart_7_C",
        "Amount": 0,
        "RemainingCost": 500,
        "TotalCost": 500
      },
      {
        "Name": "Magnetic Field Generator",
        "ClassName": "Desc_SpaceElevatorPart_6_C",
        "Amount": 0,
        "RemainingCost": 500,
        "TotalCost": 500
      },
      {
        "Name": "Thermal Propulsion Rocket",
        "ClassName": "Desc_SpaceElevatorPart_8_C",
        "Amount": 0,
        "RemainingCost": 250,
        "TotalCost": 250
      },
      {
        "Name": "Nuclear Pasta",
        "ClassName": "Desc_SpaceElevatorPart_9_C",
        "Amount": 100,
        "RemainingCost": 0,
        "TotalCost": 100
      }
    ],
        
![image](https://github.com/user-attachments/assets/f796011a-aea5-43af-8e1d-0d442f53ef6e)

**Comment by derpierre65** [comment]
looks good for me

**Comment by porisius** [comment]
Fixed in 1.1.0
